### PR TITLE
[CPDP-1254] - Fixed message format and Residents Relief No behaviour

### PR DESCRIPTION
**Pull request by modivilioglu**

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/finance/MoneyUtils.scala [hunk 1]
```diff
@@ -56,9 +56,13 @@ object MoneyUtils {
 
   def validateValueIsLessThan(key: String, limitInclusive: AmountInPence, errorMessageKey: => String)(
     value: BigDecimal
-  ): Either[FormError, BigDecimal] =
+  )(implicit messages: Messages): Either[FormError, BigDecimal] =
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/finance/MoneyUtils.scala]: is this what was breaking it? What does this do here?

Comment (modivilioglu) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/finance/MoneyUtils.scala]: Nope, removed.

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/finance/MoneyUtils.scala [hunk 2]
```diff
@@ -58,7 +57,11 @@ object MoneyUtils {
     value: BigDecimal
   ): Either[FormError, BigDecimal] =
     if (value <= limitInclusive.inPounds()) Right(value)
-    else Left(FormError(key, errorMessageKey, List(limitInclusive.inPounds().toString())))
+    else {
+      Left(
+        FormError(key, errorMessageKey, List(limitInclusive.inPounds().toString()))
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/finance/MoneyUtils.scala]: the `List` in the error message might originate from here - what happens if you don't wrap the value in a list?

Comment (modivilioglu) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/finance/MoneyUtils.scala]: Yes, it is already fixed. I was giving the wrong key, thanks

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**